### ISSUE TITLE
[Bug] Aria label for series-custom is displayed in wrong language

### ISSUE THREAD

**Issue by alexjcook** [apache/echarts#18054]
### Version

5.4.1

### Link to Minimal Reproduction

https://echarts.apache.org/examples/en/editor.html?c=custom-bar-trend

### Steps to Reproduce

1. Visit the echarts example for Custom Bar Trend:
https://echarts.apache.org/examples/en/editor.html?c=custom-bar-trend
2. Toggle "Decal Pattern" to true (which automatically sets `aria.label.enabled` to `true`)
3. Use firefox accessibility tab to view the generated aria description for the chart

### Current Behavior

The generated aria description starts with
"This is a chart. It consists of 8 series count. The 0 series is a **自定义图** representing trend."

### Expected Behavior

The aria description should be generated in the user's local language.
"This is a chart. It consists of 8 series count. The 0 series is a **custom series** representing trend."

### Environment

```markdown
- OS: Windows 10
- Browser: Firefox 107, 64-bit
- Framework: N/A
```

### Any additional comments?

Ideally for a custom series we could define our own aria description to describe what is rendered, instead of "custom". 
For the above example the custom series is rendered as a line, so it would be great if we could configure an aria-related option on the series so the following description could be generated
"This is a chart. It consists of 8 series count. The 0 series is a **line** representing trend."

**Comment by plainheart** [comment]
@Ovilia It seems the fallback series type name is hard-coded as `自定义图`. It should be able to be translated in each locale file.
https://github.com/apache/echarts/blob/master/src/visual/aria.ts#L267
